### ISSUE TITLE
About reproduction

### ISSUE THREAD

**Issue by Afleve** [yichen928/ActiveFT#6]
Thanks for ur contribution.

I could't reproduce these results in  Image Classification where trained on a single RTX 4090.
I tried 0.5% getting Acc@1 59.240 Acc@5 94.210 and tried 2% getting acc@78 in CIFAR10.  And I tried hlaf lr getting the similar results（0.5->Acc@1 58.720 Acc@5 93.920 loss 1.277）. So the experiment shows that I have to have two Gpus in order to achieve the results in this paper?
My finetuning key parameters are set as follows:
`
parser.add_argument('--batch-size', default=512, type=int)

parser.add_argument('--epochs', default=1000, type=int)  # modify from 300 to 1000

parser.add_argument('--opt', default='SGD', type=str, metavar='OPTIMIZER',
                        help='Optimizer (default: "adamw"')  # modify from adamw to sgd

parser.add_argument('--clip-grad', type=float, default=2.0, metavar='NORM',
                        help='Clip gradient norm (default: None, no clipping)')

parser.add_argument('--momentum', type=float, default=0.9, metavar='M',
                        help='SGD momentum (default: 0.9)')

parser.add_argument('--weight-decay', type=float, default=1e-4,
                        help='weight decay (default: 0.05)')  # modify from 0.05 to 1e-4

parser.add_argument('--sched', default='cosine', type=str, metavar='SCHEDULER',
                        help='LR scheduler (default: "cosine"')

parser.add_argument('--lr', type=float, default=1e-3, metavar='LR',
                        help='learning rate (default: 5e-4)')  # modify from 5e-4 to 0.01

parser.add_argument('--data-set', default='CIFAR10SUBSET',
                        choices=['CIFAR10', 'CIFAR100', 'IMNET', 'INAT', 'INAT19', 'CIFAR10SUBSET', 'CIFAR100SUBSET', 'IMNETSUBSET'],
                        type=str, help='Image Net dataset path')

parser.add_argument('--resume', default='/home/ubuntu20/.cache/torch/hub/checkpoints/dino_deitsmall16_pretrain.pth', help='resume from checkpoint')

parser.add_argument('--subset_ids', default='features/CIFAR10_train_ActiveFT_cosine_temp_0.07_lr_0.001000_scheduler_none_iter_300_sampleNum_250.json', type=str, help='a json file of sampled dataset subset')
`
Hope for ur reply.

**Comment by Afleve** [comment]
I trianed on 2 GPUs getting it. Thanks for ur contributions.

**Comment by Afleve** [comment]
I noticed in the supplementary materials of the paper that the fine-tuning parameters of CIFAR10 and CIFAR100 are consistent, but during my testing, CIFAR10 can reproduce the results directly, but CIFAR100 cannot. Above are the results from the paper, and below are the results from my experiment. Hope for ur complete fine-tuning parameters or random seeds.
![1731053066986](https://github.com/user-attachments/assets/66e796fd-537e-4776-9d76-3678ad7d018d)

**Comment by yichen928** [comment]
Hi thanks for your interest. The finetuning parameters on CIFAR10 and CIFAR100 are indeed consistent. You should also use two GPUs for CIFAR100 to reproduce our results. 

I think you may start from random sampling to check whether the problem comes from data selection or random sampling. The results on CIFAR100 are indeed unstable, but your results look much lower than normal cases.

**Comment by Afleve** [comment]
Thanks for your reply. I have tried several times and found that different seeds using the same sub tag are very unstable. I'm curious about what caused such unstable results.

**Comment by yichen928** [comment]
I am also not sure about this, but goods news is that results on CIFAR10 and ImageNet are both much more stable.

**Comment by Afleve** [comment]
u r right!
Thanks for ur reply.
